docs/content/how-to/azure/backup-and-restore-etcd-snapshot.md

```
@@ -262,8 +262,246 @@ az storage account delete \
+    --hc-name my-hosted-cluster \
+    --hc-namespace clusters \
+    --use-etcd-snapshot
+```
+
+The `--use-etcd-snapshot` mode creates an `HCPEtcdBackup` CR and excludes PV-related resources from the Velero backup. This mode requires the `HCPEtcdBackup` feature gate and Azure Blob Storage configured as described in the [Setup](#setup) section.
+
+### Restoring with the CLI
+
+Before restoring, clean up the existing HostedCluster:
+
+```bash
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)